packages/ui/package.json

```diff
@@ -42,8 +42,10 @@
     "@babel/runtime": "^7.17.9",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.5",
     "@radix-ui/react-alert-dialog": "^1.0.4",
+    "@radix-ui/react-collapsible": "^1.0.3",
     "@radix-ui/react-dialog": "^1.0.4",
     "@radix-ui/react-dropdown-menu": "^2.0.5",
+    "@radix-ui/react-icons": "^1.3.0",
```

Comment:
This lib is not used

Reply:
yeah, realized you have custom icons for arrows so I exchanged, its being deleted.

Btw, why did you chose not to use these icon pack? too slims?

Reply:
removed 👍🏽